examples/addsvc/client/grpc/client.go

```diff
@@ -4,20 +4,32 @@ import (
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 
-	"github.com/go-kit/kit/examples/addsvc/pb"
+	"github.com/go-kit/kit/endpoint"
 	"github.com/go-kit/kit/examples/addsvc/server"
 	"github.com/go-kit/kit/log"
+	grpctransport "github.com/go-kit/kit/transport/grpc"
 )
 
 // New returns an AddService that's backed by the provided ClientConn.
 func New(ctx context.Context, cc *grpc.ClientConn, logger log.Logger) server.AddService {
-	return client{ctx, pb.NewAddClient(cc), logger}
+	var replies replies
+	sumClient := grpctransport.NewClient(cc, "Add", "sum", encodeSumRequest, decodeSumResponse, replies.sumReply)
+	concatClient := grpctransport.NewClient(cc, "Add", "concat", encodeConcatRequest, decodeConcatResponse, replies.concatReply)
```

Comment:
Unless I'm missing something, you can pass concrete types directly to the NewClients, and remove the `replies` struct...